devicetypes/smartthings/zigbee-button.src/zigbee-button.groovy

```diff
@@ -91,6 +105,27 @@ def parse(String description) {
     }
 }
 
+def batteyHandler(String description){
+	def descMap = zigbee.parseDescriptionAsMap(description)
+	def map = [:]
+	if (descMap?.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER && descMap.commandInt != 0x07 && descMap.value) {
```

Comment:
Instead of just checking that this isn't `0x07` you should check that the command is either `0x01` for the read attr response or `0x0A` for a report attribute.  You should also add a check here that the `attrInt` is what you expect.  I'm guessing in this case from the usage below that it should be `0x0021` as it appears to be the battery percentage remaining.